src/scripts/__tests__/googleFormsToJson.spec.ts

```diff
@@ -6,13 +6,15 @@ import fs from 'fs'
 import path from 'path'
 
 import { googleFormsToJson } from '../googleFormsToJson'
+import { mockedParsedForm } from '../__mocks__/mockedParsedForm'
 
 jest.mock('axios')
 const mockedAxios = axios as jest.Mocked<typeof axios>
 
 describe('googleFormsToJson', () => {
   const FORM_URL =
     'https://docs.google.com/forms/d/e/1FAIpQLSe5U3qvg8WHs4nkU-e6h2RlAD7fKoCkou6HO2w2-tXYIA_F8g/viewform'
+  const SHORTENED_FORM_URL = ''
```

Comment:
We need the shortened url of your Test Form to complete the test case.

Reply:
I am not familiar how you shorten urls for google forms 😅  but since we are mocking the call using axios on this test I think we can fill this `SHORTENED_FORM_URL` with any url of your choice

Reply:
I think it is best to use the same form to check that the two responses are identical.

![image](https://user-images.githubusercontent.com/25715495/132228169-6cbe42bb-ee6c-4ad9-bfd2-86c85c09fb7b.png)

Let me know if you have any questions about how to get it 😃

Reply:
Here it is https://forms.gle/vXSsfKSvpVJc3NMg9 🙏